js/modules/debug.js

```diff
@@ -81,7 +88,7 @@ const createRandomMessage = ({ conversationId } = {}) => {
 
   const hasAttachment = Math.random() <= ATTACHMENT_SAMPLE_RATE;
   const attachments = hasAttachment
-    ? [createRandomInMemoryAttachment()] : [];
+    ? [await createRandomInMemoryAttachment()] : [];
```

Comment:
It's still weird to me that you can jam await into any part of an expression like this.

Reply:
Haha, yes, it takes some getting used to. But the cool thing is that the code is not blocking compared to languages such as Python, etc. where async code is harder 😄